CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+## [0.41.0] - 2022-09-22
```

Comment:
revert the changelog changes, I think we can wait until next week's generation to release this